manifests/b/Borvid/HttpMasterExpress/5.5.0/Borvid.HttpMasterExpress.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=AUSU.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Borvid.HttpMasterExpress
+PackageVersion: 5.5.0
+MinimumOSVersion: 10.0.0.0
+InstallerType: msi
+Installers:
+- Architecture: neutral
+  InstallerUrl: https://www.httpmaster.net/downloads/httpmaster_exp_5.5.0.msi
+  InstallerSha256: D5D120894E12B0CA9691BF9717EE4528E87DB3258FC1917610D94CBC8369120E
+  ProductCode: '{4FE8DB79-7F6E-4B2B-BBFA-FC7007E6A2AF}'
```

```suggestion
  ProductCode: '{49A2448A-CD0E-4601-83A3-0E7B8EB8C93A}'
```
Must match control panel